src/Builder/ArgumentBuilder/ArgumentBuilder.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
```

Comment:
All classes in integration project use strict types. So it will be good if you also add here:

`declare(strict_types=1);`

Reply:
++